lib/rules/flag-role.json

```diff
@@ -0,0 +1,13 @@
+{
+	"id": "flag-role",
+	"selector": "[role=\"none\"], [role=\"presentation\"]",
+	"tags": ["cat.aria"],
+	"metadata": {
+		"description": "Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger.",
+		"help": "Elements of role none or presentation should be flagged"
+	},
+	"preload": false,
+	"all": ["is-element-focusable", "has-global-aria-attribute"],
```

Comment:
So this will pass the rule if the element is both focusable and has a global aria attribute. Instead it should fail the rule, so I believe these should be in the `none` array.